Solutions/Azure Active Directory/Analytic Rules/AccountCreatedandDeletedinShortTimeframe.yaml

```diff
@@ -42,15 +42,15 @@ query: |
   | extend CreatedByUser = tostring(InitiatedBy.user.userPrincipalName), CreatedByIPAddress = tostring(InitiatedBy.user.ipAddress)
```

Comment:
In this one we have some fixed position parameters such as `TargetResources[0]` - we should replace this with parse where statements as those positional indicators aren't reliable. In addition this query has some duplicate code we could functionize to make it easier to read

Reply:
Removed TargetResource[0]
For constructing a function for code readability, I will reconsider.